app/src/main/java/org/oppia/android/app/options/AppLanguageActivity.kt

```diff
@@ -43,6 +43,13 @@ class AppLanguageActivity : InjectableAppCompatActivity() {
       intent.putExtra(APP_LANGUAGE_PREFERENCE_SUMMARY_VALUE_EXTRA_KEY, summaryValue)
       return intent
     }
+
+    fun getIntentKey(): Array<String> {
```

Comment:
this is not what exactly we are looking for. 

Create separate functions for both the items and name the functions accordingly with that function return.

Reply:
@anandwana001 is it not better if we have just one function and return the array of strings? In the future, if suppose some other key is added then we need to create one more function?

Reply:
Separate function sounds good. Passing by array is something we need one param, but getting a everything and then extracting out one.

Reply:
@anandwana001 Is this okay? I did the changes.